receiver/statsdreceiver/testdata/config.yaml

```diff
@@ -8,8 +8,12 @@ receivers:
     timer_histogram_mapping:
       - statsd_type: "histogram"
         observer_type: "gauge"
```

Comment:
Should this be updated to histogram?

Reply:
I had missed some testing for invalid configuration. This is improved to verify histogram configs are only present with histogram observer type, etc.